app/assets/stylesheets/navbar.scss

```diff
@@ -0,0 +1,69 @@
+
+a:hover, a:visited, a:link, a:active
```

Comment:
I would recommend to use `https://github.com/brigade/scss-lint` for making sure that your scss code is consistent. Here are some overriden rules that we use in SS:
```
# .scss-lint.yml

linters:
  NestingDepth:
    max_depth: 5

  SelectorDepth:
    max_depth: 5

  QualifyingElement:
    enabled: false
```

Reply:
checked, just fixed scss consistency and override rules based on what you use